source/tutorials/jupyter/jupyhai/index.rst

```diff
@@ -11,94 +11,74 @@ In this tutorial, we will use Jupyhai, to run a Valohai execution from your loca
-* Open the new Notebook
-* Click on **Settings** on the toolbar
-* Login with your Valohai credentials
-* Update the Docker image to ``valohai/notebook:tensorflow-2.5.0``
-* Close the **Settings** tab
-* Add ``print('Hello Valohai Notebooks!')`` to the first cell
-* Click on ``Run remote`` to run the notebook remotely
-* View the logs from the execution by clicking on the gizmo on the right side of the page (e.g. ``#1``)
+Create a new directory on your machine and place in there the two files that you downloaded above. Then open JupyterLab from that directory by running ``jupyter-lab`` on your command line.
 
-.. tip::
+Finally, go to the Valohai web app and create a new project.
 
-   All Valohai notebook executions have to be based on a custom Valohai Docker image ``valohai/pypermill``. You can build your own images for Notebook executions, as long as they're based on ``valohai/pypermill``
+Run your notebook on Valohai
+------------------------------
 
-   Check out our prebuilt set of `Docker images for the most popular Notebook executions on Valohai </howto/docker/popular-notebook-images/>`_
+Let's start by running your existing notebook in Valohai.
 
+You should have your Jupyter environment open with only two files, the notebook and the ``churn.csv`` file.
 
+1. Update the notebook's second cell to read the csv file from the correct path:
 
-.. video:: /_static/videos/jupyter-login-launch.mp4
-   :autoplay:
-   :width: 600
-
-
-
-.. admonition:: Open a Notebook from a previous execution
-   :class: tip
-
-   Each of the colored gizmos on the right side of the page signify a single Valohai execution. You can click on any of the completed executions and select ``Notebook`` to inspect the Notebook version that was used to run the execution.
-
-..
-
-Valohai 101 tutorial
-----------------------
+   .. code-block:: python
+      :linenos:
+      :emphasize-lines: 1
 
-Follow our Fundamentals tutorial with valohai-utils to learn how to interact with Valohai inputs, outputs, metadata, and parameters. 
+      churn_df = pd.read_csv('churn.csv')
+      col_names = churn_df.columns.tolist()
 
-`Valohai Fundamentals with valohai-utils </tutorials/learning-paths/fundamentals/valohai-utils/>`_ 
+      print "Column names:"
+      print col_names
 
+      to_show = col_names[:6] + col_names[-6:]
 
-.. hint::
+      print "\nSample data:"
+      churn_df[to_show].head(6)
 
-   Are you looking for the old Notebook tutorial that uses ``tags`` to define inputs and parameters? Find it `here </tutorials/jupyter/jupyhai/jupyhai-with-tags/>`_.
+2. Click on the **Run Remote** button and start by logging in with your Valohai username/password or API Token (if you have MFA setup).
+3. Choose your project from the dropdown menu and optionally choose a different execution environment.
+4. Change the default Docker image to ``valohai/notebook:sklearn-1.0``. This is a custom Valohai hosted image that contains sklearn, matplotlib, pandas, and numpy.
```

Comment:
Add here 5. How to then actually start the execution (do you need to click again on the run remote button after logging in) and mention that you can change the project, environment etc. by opening the Valohai Settings tab from the Settings menu.

Reply:
Ah crap, this was a work in progress doc that ended up in this commit. Will remove